openedx/features/partners/tests/factories.py

```diff
@@ -0,0 +1,23 @@
+from factory.django import DjangoModelFactory
+
+from openedx.features.partners.models import Partner, PartnerUser
+from lms.djangoapps.onboarding.models import Organization
+
+
+class PartnerFactory(DjangoModelFactory):
+    """ Partner factory."""
```

Comment:
Let's add proper documentation, `""" Partner factory."""` is only stating the obvious

Reply:
@zahid-arbisoft its a factory class for partner.What extra comment could we possibly add?Should we explain it here where this will be used?

Reply:
I thinks its purpose will be more suffice.